AWS Project/index.html

```diff
@@ -0,0 +1,300 @@
+           <input id="act_num3" class="form-control" type="text">
+       </div>
+
+      </div>
+      <!-- <div><p></p></div> -->
+      <a class="nav-link">
+        <button id="last_motion_bt" class="btn btn-sm btn-outline-light" type="button" name="button">GET DATA</button>
+      </a>
+      </div>
+
+      <div class="col-lg-2 col-sm-2"></div>
+    </div>
+
+
+    <div class="row">
+        <div class="ol-lg-12"><p></p></div>
+    </div>
+
+    <div class="row">
+        <div class="ol-lg-12"><p></p></div>
+    </div>
+
+    <div class="row">
+      <div class="col-lg-2 col-sm-2"></div>
+
+        <div class="col-lg-8 col-sm-8 boxy">
+        <!-- <div class="form-group"> -->
+        <!--
+        <div class="form-group pmd-textfield pmd-textfield-filled pmd-textfield-floating-label">
+        	  <label class="control-label">Text Area</label>
+            <textarea required class="form-control"></textarea>
+        </div>
+        </div>
+        -->
+
+        <div class="form-group pmd-textfield pmd-textfield-filled pmd-textfield-floating-label">
+           <p></p>
+	         <label for="filledfloating">Enter master key to turn off all devices</label>
+	         <input id="filledfloating" class="form-control" type="text">
+       </div>
+
+       <a class="nav-link">
+        <button class="btn btn-sm btn-outline-light" type="button" name="button">SUBMIT</button>
+       </a>
+
+     </div>
+
+
+     <div class="col-lg-2 col-sm-2"></div>
+   </div>
+
+   <div class="row">
+     <div class="col-lg-12 col-md-4"><p></p></div>
+     <div class="col-lg-12 col-md-4"><p></p></div>
+   </div>
+
+   <div class="row">
+     <div class="col-lg-2 col-sm-2"></div>
+
+       <div class="col-lg-8 col-sm-8 boxy">
+       <div class="form-group pmd-textfield pmd-textfield-filled pmd-textfield-floating-label">
+          <p></p>
+          <label for="filledfloating">Enter master key to turn on all devices</label>
+          <input id="filledfloating" class="form-control" type="text">
+      </div>
+
+      <a class="nav-link">
+       <button class="btn btn-sm btn-outline-light" type="button" name="button">SUBMIT</button>
+      </a>
+
+    </div>
+
+
+    <div class="col-lg-2 col-sm-2"></div>
+  </div>
+
+   <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+
+    <script src="app.js"></script>
+
+
+
+</body>
+</html>
```

Comment:
I dont see any changes that I mentioned in last call... can you ask pranav for the changes?